src/main/resources/db/changelog/ddl/0010_create_temporary_certificate_tables.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="nathan.russell@valtech.com" id="0010-EIP1-3610-create-temporary-certificate-table" context="ddl">
+        <createTable tableName="temporary_certificate">
+            <column name="id" type="uuid">
+                <constraints nullable="false" primaryKey="true" unique="true"
+                             primaryKeyName="temporary_certificate_pkey"/>
+            </column>
+            <column name="certificate_number" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="gss_code" type="varchar(80)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="source_type" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="source_reference" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="application_reference" type="varchar(255)"/>
+            <column name="certificate_template_version" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issuing_authority" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="english_ero_id" type="uuid">
```

Comment:
I wonder if we need these as they do not appear on the temporary certificate

Reply:
Agreed; gone 😁